modules/comments/api/getExecutiveComments.ts

```diff
@@ -0,0 +1,46 @@
+import { connectToDatabase } from 'lib/api/utils';
```

Comment:
aggregates the fetch comments logic for executive, theres' one similar for polls